benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlTpchSchema.java

```diff
@@ -0,0 +1,132 @@
+    primaryKeyMap.put("region", new LinkedList<>(Arrays.asList("r_regionkey")));
+    primaryKeyMap.put("supplier", new LinkedList<>(Arrays.asList("s_suppkey")));
+
+    foreignKeyMap = new LinkedHashMap<>();
+    foreignKeyMap = ImmutableMap
+        .of("customer", ImmutableMap.of("c_nationkey", (ImmutableMap.of("nation", "n_nationkey"))));
+    foreignKeyMap = ImmutableMap
+        .of("lineitem", ImmutableMap.of("l_orderkey", (ImmutableMap.of("orders", "o_orderkey")),
+            "l_partkey", (ImmutableMap.of("part", "p_partkey")),
+            "l_suppkey", (ImmutableMap.of("supplier", "s_suppkey"))));
+    foreignKeyMap = ImmutableMap
+        .of("nation", ImmutableMap.of("n_regionkey", (ImmutableMap.of("region", "r_regionkey"))));
+    foreignKeyMap = ImmutableMap
+        .of("orders", ImmutableMap.of("o_custkey", (ImmutableMap.of("customer", "c_custkey"))));
+    foreignKeyMap = ImmutableMap
+        .of("partsupp", ImmutableMap.of("ps_partkey", (ImmutableMap.of("part", "p_partkey")),
+            "ps_suppkey", (ImmutableMap.of("supplier", "s_suppkey"))));
+    foreignKeyMap = ImmutableMap
+        .of("supplier", ImmutableMap.of("s_nationkey", (ImmutableMap.of("nation", "n_nationkey"))));
+
+    schemaMap = new LinkedHashMap<>();
+
+    LinkedHashMap customerArgs = new LinkedHashMap<>();
+    customerArgs.put("c_custkey", "bigint");
+    customerArgs.put("c_name", "text");
+    customerArgs.put("c_address", "text");
+    customerArgs.put("c_nationkey", "bigint");
+    customerArgs.put("c_phone", "text");
+    customerArgs.put("c_acctbal", "decimal");
+    customerArgs.put("c_mktsegment", "text");
+    customerArgs.put("c_comment", "text");
+    schemaMap.put("customer", customerArgs);
+
+    LinkedHashMap lineitemArgs = new LinkedHashMap<>();
+    lineitemArgs.put("l_orderkey", "bigint");
+    lineitemArgs.put("l_partkey", "bigint");
+    lineitemArgs.put("l_suppkey", "bigint");
+    lineitemArgs.put("l_linenumber", "int");
+    lineitemArgs.put("l_quantity", "decimal");
+    lineitemArgs.put("l_extendedprice", "decimal");
+    lineitemArgs.put("l_discount", "decimal");
+    lineitemArgs.put("l_tax", "decimal");
+    lineitemArgs.put("l_returnflag", "text");
+    lineitemArgs.put("l_linestatus", "text");
+    lineitemArgs.put("l_shipdate", "date");
+    lineitemArgs.put("l_commitdate", "date");
+    lineitemArgs.put("l_receiptdate", "date");
+    lineitemArgs.put("l_shipinstruct", "text");
+    lineitemArgs.put("l_shipmode", "text");
+    lineitemArgs.put("l_comment", "text");
+    schemaMap.put("lineitem", lineitemArgs);
+
+    LinkedHashMap nationArgs = new LinkedHashMap<>();
+    nationArgs.put("n_nationkey", "bigint");
+    nationArgs.put("n_name", "text");
+    nationArgs.put("n_regionkey", "bigint");
+    nationArgs.put("n_comment", "text");
+    schemaMap.put("nation", nationArgs);
+
+    LinkedHashMap ordersArgs = new LinkedHashMap<>();
+    ordersArgs.put("o_orderkey", "bigint");
```

Comment:
Please create constants for "bigint", "text", etc.

Reply:
done